deploy-frontend-internal.sh

```diff
@@ -26,9 +26,9 @@ docker run --detach \
     -e SRC_FRONTEND_INTERNAL=sourcegraph-frontend-internal:3090 \
     -e REPO_UPDATER_URL=http://repo-updater:3182 \
     -e REPLACER_URL=http://replacer:3185 \
-    # remove after 3.15
+    `# remove after 3.15` \
```

Comment:
I am surprised this doesn't cause the `docker` command to break or complain about an extra argument?

Reply:
On the contrary, this is required to fix the very issue you're pointing out. Bash doesn't permit comments in the midst of multi-line constructions, and the backtick syntax is a way to "trick" it into interpreting this as just another line.